Make readme not so misleading (#15)
invenia/JLSO.jl: README.md

[![Build Status](https://ci.appveyor.com/api/projects/status/github/invenia/JLSO.jl?svg=true)](https://ci.appveyor.com/project/invenia/JLSO-jl)
[![Codecov](https://codecov.io/gh/invenia/JLSO.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/invenia/JLSO.jl)

JLSO is a storage container for serialized Julia objects.
At the top-level it is a BSON file,
where it stores metadata about the system it was created on as well as a collection of objects (the actual data).

Depending on configuration, those objects may themselves be stored as BSON sub-documents,
or in the native Julia serialization format (default).
It is fast and efficient to load just single objects out of a larger file that contains many objects.

> The metadata (always stored in BSON) includes the Julia version and the versions of all packagers installed.

Reviewer: ```suggestion
The metadata (always stored in BSON) includes the Julia version and the versions of all packages installed.
```

> This means in the worst case you can installed everything again and replicate your system.

Reviewer: ```suggestion
This means in the worst case you can install everything again and replicate your system.
```

> (Extreme worst case senario, using a BSON reader from anyther programming language).

Reviewer: ```suggestion
(Extreme worst case scenario, using a BSON reader from another programming language).
```

> Note: that if the amount of data you have to store is very small, relative to the metadata about your enviroment.

Reviewer: ```suggestion
Note: If the amount of data you have to store is very small, relative to the metadata about your environment, then it is a pretty suboptimal format.
```

> Note: that if the amount of data you have to store is very small, relative to the metadata about your enviroment.

Reviewer 2: ```suggestion
Note: that if the amount of data you have to store is very small, relative to the metadata about your environment.
```

> Then is is a pretty suboptimal format.

Reviewer: ```suggestion
```

Author: Why?What is the harm of end of file newlines?
Atom inserts them automatically...

Reviewer: i meant "applying this suggestion might leave two newlines at the end of the file instead of one"

Author: Oh right gotcha

Author: Pretty sure it does not
